.github/workflows/CI_publish_snapshots.yml

```diff
@@ -28,7 +28,10 @@ jobs:
                     packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
                     packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
                 run: |
-                    ./gradlew clean build -x createJavadoc --scan publish --continue --rerun-tasks
+                    ./gradlew clean build createCodeCoverageReport -x createJavadoc --scan publish --continue --rerun-tasks
```

Comment:
Should we do the same for release work flow as well?
Or is it enough to have coverage for current sourcecode

Reply:
IMO, having the coverage for master merges and PRs would suffice. This PR only covers the master commits. in terms of the coverage in PRs, since the tests are split into multiple workflows, we need to figure out a way to get the aggregated coverages.